ambari-server/src/main/java/org/apache/ambari/server/controller/internal/RequestResourceProvider.java

```diff
@@ -851,6 +853,7 @@ private Resource getRequestResource(RequestEntity entity, String clusterName,
             hostRoleStatusCounters.get(HostRoleStatus.QUEUED), requestedPropertyIds);
     setResourceProperty(resource, REQUEST_COMPLETED_TASK_CNT_ID,
             hostRoleStatusCounters.get(HostRoleStatus.COMPLETED), requestedPropertyIds);
+    setResourceProperty(resource, REQUEST_USER_NAME, entity.getUserName(), requestedPropertyIds);
```

Comment:
UI should make sure to handle null as something like Unknown user for readability.